ai_genomics/pipeline/gtr/make_gtr_projects.py

```diff
@@ -0,0 +1,177 @@
+# Script to explore AI definition
+
+import boto3
+import logging
+import pandas as pd
+from collections import Counter
+import re
+from toolz import pipe
+
+from ai_genomics.pipeline.gtr.gtr_utils import fetch_gtr
+from ai_genomics import config, PROJECT_DIR
+
+GTR_INPUTS_DIR = PROJECT_DIR / "inputs/data/gtr"
+GTR_INPUTS_DIR.mkdir(exist_ok=True)
+
+GTR_OUTPUTS_DIR = PROJECT_DIR / "outputs/data/gtr"
+GTR_OUTPUTS_DIR.mkdir(exist_ok=True)
+
+GTR_PROJ_NAME = "gtr_ai_genomics_projects.csv"
+
+KEEP_GTR_VARS = [
+    "id",
+    "project_id",
+    "title",
+    "start",
+    "grant_category",
+    "abstract_text",
+    "potential_impact",
+]
+
+
+def camel_case_to_snake_case_col_names(df: pd.DataFrame) -> pd.DataFrame:
+    """Turn column names from camelCase to snake_case"""
+    return df.rename(columns=lambda x: re.sub(r"(?<!^)(?=[A-Z])", "_", x).lower())
+
+
+def send_output_to_s3(file_path: str, s3_destination: str):
+    """TODO: PUT THIS IN UTILS"""
+    s3 = boto3.resource("s3")
+    (s3.Bucket("ai-genomics").upload_file(file_path, f"outputs/{s3_destination}"))
+
+
+if __name__ == "__main__":
+    logging.info("Reading GtR data")
+    projects = fetch_gtr("gtr_projects-projects")
+    topics = fetch_gtr("gtr_projects-topic")
```

Comment:
I had issues running this script so needed to:

```suggestion
    topics = pd.DataFrame(fetch_gtr("gtr_projects-topic"))
```